omod_1.x/src/main/webapp/bill.jsp

```diff
@@ -17,8 +16,8 @@
 <openmrs:require allPrivileges="<%=PrivilegeWebConstants.BILL_PAGE_PRIVILEDGES%>" otherwise="/login.htm" redirect="/module/openhmis/cashier/bill.form" />
 
 <%-- Force our newer jquery version to load first --%>
-<openmrs:htmlInclude file="/moduleResources/openhmis/backboneforms/js/lib/jquery.js" />
 <%@ include file="/WEB-INF/template/header.jsp"%>
+<openmrs:htmlInclude file="/moduleResources/openhmis/backboneforms/js/lib/jquery.js" />
```

Comment:
Isn't this change going to reverse the fix making our version of jquery load first?